packages/records/lib/models/records.ts

```diff
@@ -20,6 +20,7 @@ import { Err, Ok, retry, stringToHash } from '@nangohq/utils';
 import type { Result } from '@nangohq/utils';
 import type { Knex } from 'knex';
 import { Cursor } from '../cursor.js';
+import merge from 'lodash-es/merge.js';
```

Comment:
not a huge fine of lodash. It is huge and haven't seen a release in years. If we don't want to rollout our own merge function can we find a smaller lib?

Reply:
Yep, fair. I used it because we had lodash elsewhere, but I'll dig around for a better option.

Reply:
See 51b0cd166 - switched to using `@fastify/deepmerge`. It's probably better overall since it'll let us tweak array merging as well.